src/components/organisms/Tickets.tsx

```diff
@@ -0,0 +1,77 @@
+import {useState} from 'react'
+import Ticket from '../molecules/Ticket'
+import img from '../../assets/companies/s7.png'
+
+type ITicket = {
+  price: string
+  route: string
+  routeTime: string
+  lenghtTime: string
+  transfer: string
+  transferAirport?: string
+}
+
+export default function Tickets() {
+  const [tickests] = useState<ITicket[]>([
+    {
+      price: '13 400',
+      route: 'MOW – HKT',
+      routeTime: '10:45 – 08:00',
+      lenghtTime: '21ч 15м',
+      transfer: 'HKG, JNB'
+    },
+    {
+      price: '13 400',
+      route: 'MOW – HKT',
+      routeTime: '10:45 – 08:00',
+      lenghtTime: '21ч 15м',
+      transfer: 'HKG, JNB'
+    },
+    {
+      price: '13 400',
+      route: 'MOW – HKT',
+      routeTime: '10:45 – 08:00',
+      lenghtTime: '21ч 15м',
+      transfer: 'HKG, JNB'
+    },
+    {
+      price: '13 400',
+      route: 'MOW – HKT',
+      routeTime: '10:45 – 08:00',
+      lenghtTime: '21ч 15м',
+      transfer: 'HKG, JNB'
+    },
+    {
+      price: '13 400',
+      route: 'MOW – HKT',
+      routeTime: '10:45 – 08:00',
+      lenghtTime: '21ч 15м',
+      transfer: 'HKG, JNB'
+    },
+    {
+      price: '13 400',
+      route: 'MOW – HKT',
+      routeTime: '10:45 – 08:00',
+      lenghtTime: '21ч 15м',
+      transfer: 'HKG, JNB'
+    }
+  ])
+
+  return (
+    <div className="tickets__list">
+      {tickests.map((t, idx) => {
+        return (
+          <Ticket
+            key={idx + t.lenghtTime}
```

Comment:
[Плохая практика](https://medium.com/geekculture/reactjs-why-index-as-a-key-is-an-anti-pattern-4b9dc6ef0067) создавать `key` на основе `idx`, нужно подобрать независимый уникальный ключ.

Так как возможна ситуация, когда у нас есть билеты с одинаковым времен полёта.

Я решил использовать путь и время, но и так в теории возможна коллизия

```suggestion
            key={t.route + t.lengthTime}
```